tests/Microsoft.Identity.Web.Test.Integration/AcquireTokenForAppIntegrationTests.cs

```diff
@@ -49,7 +49,7 @@ public class AcquireTokenForAppIntegrationTests
         {
             _output = output;
 
-            KeyVaultSecretsProvider keyVaultSecretsProvider = new KeyVaultSecretsProvider(TestConstants.BuildAutomationKeyVaultName);
+            KeyVaultSecretsProvider keyVaultSecretsProvider = new KeyVaultSecretsProvider(TestConstants.MSIDLabLabKeyVaultName);
```

Comment:
Why is this needed?

Reply:
This is where IdWeb uses the Lab api to acquire secrets from the Lab Keyvault to run client credential scenarios. This is how it acquires the "MSIDLAB4-IDLABS-APP-AzureADMyOrg-CC" secret.